docs-src/api-params.md

```diff
@@ -0,0 +1,120 @@
+  - `'domcontentloaded'` - consider operation to be finished when the `DOMContentLoaded` event is fired.
+  - `'load'` - consider operation to be finished when the `load` event is fired.
+  - `'networkidle'` - consider operation to be finished when there are no network connections for at least `500` ms.
+
+## navigation-timeout
+- `timeout` <[number]> Maximum operation time in milliseconds, defaults to 30 seconds, pass `0` to disable timeout. The default value can be changed by using the [browserContext.setDefaultNavigationTimeout(timeout)](#browsercontextsetdefaultnavigationtimeouttimeout), [browserContext.setDefaultTimeout(timeout)](#browsercontextsetdefaulttimeouttimeout), [page.setDefaultNavigationTimeout(timeout)](#pagesetdefaultnavigationtimeouttimeout) or [page.setDefaultTimeout(timeout)](#pagesetdefaulttimeouttimeout) methods.
+
+## wait-for-timeout
+- `timeout` <[number]> maximum time to wait for in milliseconds. Defaults to `30000` (30 seconds). Pass `0` to disable timeout. The default value can be changed by using the [browserContext.setDefaultTimeout(timeout)](#browsercontextsetdefaulttimeouttimeout).
+
+## input-timeout
+- `timeout` <[number]> Maximum time in milliseconds, defaults to 30 seconds, pass `0` to disable timeout. The default value can be changed by using the [browserContext.setDefaultTimeout(timeout)](#browsercontextsetdefaulttimeouttimeout) or [page.setDefaultTimeout(timeout)](#pagesetdefaulttimeouttimeout) methods.
+
+## input-no-wait-after
+- `noWaitAfter` <[boolean]> Actions that initiate navigations are waiting for these navigations to happen and for pages to start loading. You can opt out of waiting via setting this flag. You would only need this option in the exceptional cases such as navigating to inaccessible pages. Defaults to `false`.
+
+## input-force
+- `force` <[boolean]> Whether to bypass the [actionability](./actionability.md) checks. Defaults to `false`.
+
+## input-selector
+- `selector` <[string]> A selector to search for element. If there are multiple elements satisfying the selector, the first will be used. See [working with selectors](#working-with-selectors) for more details.
+
+## input-position
+- `position` <[Object]> A point to use relative to the top-left corner of element padding box. If not specified, uses some visible point of the element.
+  - `x` <[number]>
+  - `y` <[number]>
+
+## input-modifiers
+- `modifiers` <[Array]<"Alt"|"Control"|"Meta"|"Shift">> Modifier keys to press. Ensures that only these modifiers are pressed during the operation, and then restores current modifiers back. If not specified, currently pressed modifiers are used.
+
+## input-button
+- `button` <"left"|"right"|"middle"> Defaults to `left`.
+
+## input-files
+- `files` <[string]|[Array]<[string]>|[Object]|[Array]<[Object]>>
+  - `name` <[string]> [File] name **required**
+  - `mimeType` <[string]> [File] type **required**
+  - `buffer` <[Buffer]> File content **required**
+
+## input-down-up-delay
```

Comment:
Are we after full reuse? This one, for example, looks very minor, perhaps we can copy it?

Reply:
8 occurances